src/frontend/js/components/Tabs/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button } from '@openfun/cunningham-react';
+import classNames from 'classnames';
+import {
+  Dispatch,
+  PropsWithChildren,
+  SetStateAction,
+  createContext,
+  useContext,
+  useMemo,
+  useState,
+} from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Maybe } from 'types/utils';
+
+interface TabsContextProps {
+  activeTabName?: string;
+  setActiveTabName?: Dispatch<SetStateAction<Maybe<string>>>;
+}
+const TabsContext = createContext<TabsContextProps>({});
+
+const TabContextProvider = ({
+  initialActiveTabName,
+  children,
+}: PropsWithChildren & { initialActiveTabName?: string }) => {
+  const [activeTabName, setActiveTabName] = useState(initialActiveTabName);
+  const contextValue = useMemo(() => {
+    return { activeTabName, setActiveTabName };
+  }, [activeTabName]);
+
+  return <TabsContext.Provider value={contextValue}>{children}</TabsContext.Provider>;
+};
+
+const Tabs = ({
+  initialActiveTabName,
+  children,
+}: PropsWithChildren & { initialActiveTabName: string }) => {
+  return (
+    <div className="tabs">
+      <TabContextProvider initialActiveTabName={initialActiveTabName}>
+        {children}
+      </TabContextProvider>
+    </div>
+  );
+};
+
+Tabs.Header = ({ children }: PropsWithChildren) => {
+  return <div className="tabs__header">{children}</div>;
+};
+
+Tabs.Content = ({ children }: PropsWithChildren) => {
+  return <div className="tabs__content">{children}</div>;
+};
+
+interface TabProps extends PropsWithChildren {
+  name: string;
+  href?: string;
+  onClick?: () => void;
+}
+
+Tabs.Tab = ({ name, href, onClick, children }: TabProps) => {
+  const { activeTabName, setActiveTabName } = useContext(TabsContext);
+  const navigate = useNavigate();
+
+  const isActive = !!activeTabName && activeTabName === name;
```

Comment:
I think this should work ?
```suggestion
  const isActive = activeTabName === name;
```